doc/website/blog/2022-08-11-announcing-versioned-migration-authoring.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: Announcing v0.6.0 with Versioned Migration Authoring
+authors: a8m
+tags: [atlas, migrations, versioned]
+image: https://blog.ariga.io/uploads/images/posts/v0.5.0/atlas-lint.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+With the release of [v0.6.0](https://github.com/ariga/atlas/releases/tag/v0.6.0), we are
+happy to announce official support for a style of workflow for managing changes to database
+schemas that we have been experimenting with in the past months: _Versioned Migration Authoring_.
+
+### tl;dr
```

Comment:
```suggestion
### TL;DR
```
Or `TLDR`

I am not sure I like this section. It breaks the reading flow because the post reads repetitive since we read parts doubly.